magento/pwa-studio
# [doc] Release notes 10.0.0

## Description

Updates the CHANGELOG and compatibility table for 10.0.0
## Related Issue

Closes PWA-1565

## Acceptance

@davemacaulay
@ericerway 
@tkacheva 

### Verification Stakeholders

@davemacaulay
@ericerway 
@tkacheva 

### Specification



### Verification Steps


1. Open CHANGELOG
2. Verify content is accurate

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
